Tests/MongoSwiftTests/AsyncAwaitTestUtils.swift

```diff
@@ -66,8 +66,6 @@ extension MongoSwiftTestCase {
         return try await f(client)
     }
 
-    // swiftlint:disable large_tuple
```

Comment:
this is the SwiftLint bug I contributed a fix for a while back. with the upgrade this started producing a warning about an unneeded "disable" statement